File: PHP/8php 3/simpleMul.php
<?php
/*
|--------------------------------------------------------------------------
| Simple multiplication
|--------------------------------------------------------------------------
|
*/

function simpleMultiplication($n) {
   return $n * (8 + $n%2);
}


function simpleMultiplication($number) {
   return ($number % 2 == 0) ? $number * 8 : $number * 9;
}


function simpleMultiplication($a) {
   return $a * (8 + ($a & 1));
}


function simpleMultiplication($n) {
   return $n * ($n % 2 == 0 ? 8 : 9);
}


print_r(simpleMultiplication(2). ' ' . 16 . '
');
print_r(simpleMultiplication(1). ' ' . 9 . '
');
print_r(simpleMultiplication(8). ' ' . 64 . '
');
print_r(simpleMultiplication(4). ' ' . 32 . '
');
print_r(simpleMultiplication(5). ' ' . 45 . '
');

File: Python/8py 3/simple_mul.py
"""
|--------------------------------------------------------------------------
| Simple multiplication
|--------------------------------------------------------------------------
|
"""

def simple_multiplication(n) :
   return n * (8 + n%2)


simple_multiplication = lambda a : a * (8 + (a & 1))


def simple_multiplication(number) :
   return number * 8 if number % 2 == 0 else number * 9


def simple_multiplication(number) :
   return number * (8 if number % 2 == 0 else 9)


print(simple_multiplication(2), 16)
print(simple_multiplication(1), 9)
print(simple_multiplication(8), 64)
print(simple_multiplication(4), 32)
print(simple_multiplication(5), 45)